### PR TITLE
docs(operator-docs): Added MergeAll operator documentation

### PR DESCRIPTION
**Pull request by sumitarora**
Closes: https://github.com/ReactiveX/rxjs-docs/issues/67

### CODE REVIEW

**Comment by ladyleet** [comment]
Looks like the branch is out of sync @sumitarora

**Comment by sumitarora** [comment]
@ladyleet done

**Comment by ladyleet** [comment]
@sumitarora oh noes. otra vez por favor.

File: src/operator-docs/combination/mergeAll.ts [hunk 1]
```diff
@@ -1,6 +1,77 @@
-import { OperatorDoc } from '../operator.model';
+import { OperatorDoc } from "../operator.model";
```
Comment (xlozinguez) [inline src/operator-docs/combination/mergeAll.ts]: Use single quote here. This will be handled automagically once #148 is merged.

Comment (xlozinguez) [inline src/operator-docs/combination/mergeAll.ts]: Use single quote here. This will be handled automagically once #148 is merged.

**Comment by xlozinguez** [comment]
LGTM once single quote issue is fixed

File: src/operator-docs/combination/mergeAll.ts (hunk 1 above)
Comment (sumitarora) [reply src/operator-docs/combination/mergeAll.ts]: @xlozinguez It doesn't take effect as pre-commit hook change it to double quotes again. Probably would be done after #148 is merged.

**Comment by ashwin-sureshkumar** [comment]
@sumitarora please rebase to master.

**Comment by sumitarora** [comment]
@ashwin-sureshkumar done

**Pull request merged**